modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -0,0 +1,96 @@
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`domain`
+|`domain` is a DNS name serviced by the Ingress controller and is used to configure multiple features:
+
+* For the `LoadBalancerService` endpoint publishing strategy, domain is used to configure DNS records. See `endpointPublishingStrategy`.
+
+* When using a generated default certificate, the certificate will be valid for domain and its subdomains. See `defaultCertificate`.
+
+* The value is published to individual Route statuses so that end-users know where to target external DNS records.
+
+`domain` must be unique among all Ingress controllers, and cannot be updated.
+
+If empty, defaults to `ingress.config.openshift.io/cluster` `.spec.domain`.
+
+This parameter is optional.
+
+|`replicas`
+|`replicas` is the desired number of Ingress controller replicas. If not set, defaults to 2.
+
+|`endpointPublishingStrategy`
+|`endpointPublishingStrategy` is used to publish the Ingress controller endpoints to other networks, enable load balancer integrations, and so on.
+
+If not set, the default is based on `infrastructure.config.openshift.io/cluster .status.platform`:
+
+* AWS: `LoadBalancerService` (with External scope)
+* Azure: `LoadBalancerService` (with External scope)
+* GCP: `LoadBalancerService` (with External scope)
+* Libvirt: `HostNetwork`
+* Any other platform types (including None) default to `HostNetwork`.
+
+`endpointPublishingStrategy` cannot be updated.
+
+This parameter is optional.
+
+|`defaultCertificate`
+|`defaultCertificate` is a reference to a secret containing the default certificate served by the Ingress controller. When Routes do not specify their own certificate, `defaultCertificate` is used.
+
+The secret must contain the following keys and data:
+* `tls.crt`: certificate file contents
+* `tls.key`: key file contents
+
+If not set, a wildcard certificate is automatically generated and used. The certificate is valid for the Ingress controller `domain` and subdomains, and
+the generated certificate's CA will be automatically integrated with the
+cluster's trust store.
+
+The in-use certificate, whether generated or user-specified, will be automatically integrated with {product-title} built-in OAuth server.
+
+This parameter is optional.
+
+|`namespaceSelector`
+|`namespaceSelector` is used to filter the set of namespaces serviced by the
+Ingress controller. This is useful for implementing shards.
+
+If not set, the default is no filtering.
+
+This parameter is optional.
+
+|`routeSelector`
+|`routeSelector` is used to filter the set of Routes serviced by the Ingress controller. This is useful for implementing shards.
+
+If not set, the default is no filtering.
+
+This parameter is optional.
+
+|`nodePlacement`
+|`nodePlacement` enables explicit control over the scheduling of the Ingress controller.
+
+If unset, defaults are used.
+
+This parameter is optional
+
+|`tlsSecurityProfile`
```

Comment:
I need to verify, but I think this is only in 4.3. If that's the case, I'll comment it out and open a follow-up PR to add it only to 4.3.

Reply:
Yes, this is only in 4.3.

Reply:
Thank you!